docs/ConcurrentStreams.hs

```diff
@@ -723,18 +723,21 @@ import Control.Monad.Trans.Class   (MonadTrans (lift))
 -- 'fromZipAsync' type combinator can be used to switch to parallel applicative
 -- zip composition:
 --
--- This takes 7 seconds to zip, which is max (1,3) + max (2,4) because 1 and 3
--- are produced concurrently, and 2 and 4 are produced concurrently:
+-- This takes 6 seconds to zip, which is max (2 + 4, 3 + 1) because both the
+-- streams are run concurrently but the elements in the stream are produced
+-- serially. There is intra-stream concurrency but no inter-stream
+-- concurrency. Intra-stream concurrency is due to 'fromZipAsync' whereas the
+-- absence of inter-stream concurrency is due to 'fromSerial'.
 --
 -- >>> d n = delay n >> return n
--- >>> s1 = Stream.fromSerial $ d 1 <> d 2
--- >>> s2 = Stream.fromSerial $ d 3 <> d 4
+-- >>> s1 = Stream.fromSerial $ d 2 <> d 4
+-- >>> s2 = Stream.fromSerial $ d 3 <> d 1
```

Comment:
Was there some specific reason to change the values?

Reply:
The previous test fails, there is no consistent way for it to pass.
`1 + 2`
`3 + 4`
1 comes first
No way to know whether 2 or 3 comes next